requirements.txt

```diff
@@ -1,12 +1,12 @@
-numpy
-scipy
-Pillow
-cython
-matplotlib
-scikit-image
+numpy==1.16.2
+scipy==1.2.1
+Pillow==5.1.0
+cython==0.28.2
+matplotlib==2.2.2
+scikit-image==0.15.0
 tensorflow>=1.3.0
 keras>=2.0.8
```

Comment:
I think we should also lock tf and keras to specific versions

Reply:
@bdwyer2 Yes. The versions of tf and keras are specified now via https://github.com/IBM/MAX-Nucleus-Segmenter/pull/8/commits/0759b12b3c4685d8c3cc3ecc51c4e465532d2924.